admin/src/main/resources/io/buoyant/admin/js/src/router_client.js

```diff
@@ -169,8 +170,14 @@ define([
 
       function toggleClientDisplay(expand) {
         if (expand) {
+          $contentContainer.css({"border": colorBorder});
+          $headerLine.css("border-bottom", null);
+
           metricsCollector.registerListener(metricsHandler, getDesiredMetrics);
         } else {
+          $contentContainer.css({'border': null});
+          $headerLine.css({'border-bottom': colorBorder});
```

Comment:
It's my preference to separate js and css as much as possible. Instead of directly setting the border-bottom here, I'd like to add/remove a class instead. That way css for the class can remain in dashboard.css